docs/usage/v1_guide.md

```diff
@@ -83,7 +83,8 @@ based on assigned priority, with FCFS as a tie-breaker), configurable via the
 | **Decoder-only Models**     | <nobr>🚀 Optimized</nobr>                                                          |
 | **Encoder-Decoder Models**  | <nobr>🟠 Delayed</nobr>                                                            |
 | **Embedding Models**        | <nobr>🟢 Functional</nobr>                                                         |
-| **Mamba Models**            | <nobr>🚧 WIP ([PR #19327](https://github.com/vllm-project/vllm/pull/19327))</nobr> |
+| **Mamba Models**            | <nobr>🟢 Functional</nobr>                                                         |
+| **Hybrid Models**           | <nobr>🟢 Functional</nobr>                                                         |
```

Comment:
Still prefer to mark it as WIP as not all mamba models are supported. And prefer not to add a new "hybrid model" entry.